src/app/item-feed/ItemFeed.tsx

```diff
@@ -62,9 +62,10 @@ const Item = memo(function Item({ item, tag }: { item: DimItem; tag: TagValue |
 });
 
 const filteredItemsSelector = createSelector(allItemsSelector, (allItems) =>
-  allItems
-    .filter((i) => i.equipment && i.power > 0 && i.taggable)
-    .sort(acquisitionRecencyComparator)
+  _.sortBy(
```

Comment:
`_.sortBy` can be faster than `.sort` if the compareBy function is expensive to compute.